sdk/cognitiveservices/Vision.CustomVision.Training/src/Generated/CustomVisionTrainingClient.cs

```diff
@@ -682,7 +685,7 @@ public async Task<HttpOperationResponse<IList<Project>>> GetProjectsWithHttpMess
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<HttpOperationResponse<Project>> CreateProjectWithHttpMessagesAsync(string name, string description = default(string), System.Guid? domainId = default(System.Guid?), string classificationType = default(string), IList<string> targetExportPlatforms = default(IList<string>), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationResponse<Project>> CreateProjectWithHttpMessagesAsync(string name, string description = default(string), System.Guid? domainId = default(System.Guid?), string classificationType = default(string), IList<string> targetExportPlatforms = default(IList<string>), CreateProjectOptions options = default(CreateProjectOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I assume this was autogenerated but ideally new parameters are added at the end.

Reply:
Correct, all of our SDKs are autorest generated. options is the last operation parameter so it added it to the end of operation parameters, the latter params are autorest generated.